nfabackend/src/main/resources/ddl/NFADB.sql

```diff
@@ -336,26 +336,20 @@ CREATE TABLE public.nfa
  nfa_id serial PRIMARY KEY,
  NFA_NUMBER int NOT NULL,
  NFA_TYPE character varying(40),
- BEZEICHNUNG character varying(40),
  RECHTLICHE_VERBINDLICHKEIT character varying(40),
- WERT real,
+ WERT character varying(200),
  FORMULIERUNG character varying(40),
- ERKLAERUNG character varying(500),
  REFERENZ character varying(40),
  REFERENZIERTE_PROJEKTE character varying(40),
- KRITIKALITAET character varying(40),    
-DOKUMENT character varying(40));
-INSERT INTO nfa VALUES (1,1,'Type:1','Bezeichnung:1','Verbindlichkeit:1','12.22','Formulierung:1','erklaerung:1','referenz:1','referenzierte_projekt:1','kritikalitaet', 'document1');
```

Comment:
why did you remove these Values? Won't @HarishVijayamohan need them?

Reply:
I changed strucure of table and did not want to change all dummy data.